src/Microsoft.Azure.SignalR.Common/Logging/AzureSignalRLog.cs

```diff
@@ -14,6 +14,7 @@ internal static class AzureSignalRLog
         public const string StartToBroadcastMessageTemplate = "Start to broadcast message {0}.";
```

Comment:
the class name, maybe simply `Logger`

Reply:
Sounds good

Reply:
but a lot of classes already has a menber called `Logger`, they will be conflict